modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -29,10 +31,14 @@ def submit
           pdf_data = get_pdf_data
           pdf_mapper_service(form_attributes, pdf_data, target_veteran).map_claim
 
+          get_benefits_documents_auth_token unless Rails.env.test?
+
           # evss_data = evss_mapper_service(auto_claim).map_claim
-          # evss_claim = evss_service.submit(auto_claim, evss_data)
+          # evss_service.submit(auto_claim, evss_data)
 
-          get_benefits_documents_auth_token unless Rails.env.test?
+          # find_by_ssn is not the same as find_by(ssn:)
+          _file_nbr = local_bgs_service.find_by_ssn(target_veteran.ssn)&.dig(:file_nbr) # rubocop:disable Rails/DynamicFindBy
+          # benefits_doc_api.upload(auto_claim, pdf_path, file_nbr)
```

Comment:
Can you delete these commented-out lines of code?

Reply:
I prefer comments like this not be in master. However, if this feature is a WIP and you want to leave them and know they'll be removed soon, can you please add a comment as to why they're there/ what they're for?